redbot/core/utils/mod.py

```diff
@@ -222,9 +220,7 @@ async def is_admin_or_superior(
     else:
         raise TypeError("Only messages, members or roles may be passed")
 
-    if await bot.is_owner(user):
-        return True
-    if await bot.is_admin(user):
+    if user == user.guild.owner or await bot.is_owner(user) or await bot.is_admin(user):
         return True
 
     return False
```

Comment:
```suggestion
    return user == user.guild.owner or await bot.is_owner(user) or await bot.is_admin(user)
```

Looks like each of these could be simplified to just return the bool